app/Http/Middleware/RedirectIfAuthenticated.php

```diff
@@ -3,39 +3,22 @@
 namespace App\Http\Middleware;
 
 use Closure;
-use Illuminate\Contracts\Auth\Guard;
+use Illuminate\Support\Facades\Auth;
 
 class RedirectIfAuthenticated
 {
     /**
-     * The Guard implementation.
-     *
-     * @var Guard
-     */
-    protected $auth;
-
-    /**
-     * Create a new filter instance.
-     *
-     * @param  Guard  $auth
-     * @return void
-     */
-    public function __construct(Guard $auth)
-    {
-        $this->auth = $auth;
-    }
-
-    /**
      * Handle an incoming request.
      *
      * @param  \Illuminate\Http\Request  $request
      * @param  \Closure  $next
+     * @param  string|null  $guard
      * @return mixed
      */
-    public function handle($request, Closure $next)
+    public function handle($request, Closure $next, $guard = null)
     {
-        if ($this->auth->check()) {
-            return redirect('/home');
```

Comment:
Dropping /home. That was a custom route for this app. 

Sent from Outlook Mobile




On Fri, Apr 29, 2016 at 5:07 AM -0700, "Laravel Shift" <notifications@github.com> wrote:












In app/Http/Middleware/RedirectIfAuthenticated.php:
>      {
> -        if ($this->auth->check()) {
> -            return redirect('/home');



Because of the redirect('/home') to redirect('/') or the $this->auth?



—
You are receiving this because you modified the open/close state.
Reply to this email directly or view it on GitHub